assets/src/components/FrontendBlockNode/FrontendBlockNode.js

```diff
@@ -7,10 +6,12 @@ import { Component } from '@wordpress/element';
  * Be careful! Making changes in this component or in the `frontendRendered`
  * function could potentially cause block validation errors in Gutenberg.
  */
-export class FrontendBlockNode extends Component {
-	render() {
-		return <div className={ this.props.className }
-			data-render={ this.props.blockName }
-			data-attributes={ JSON.stringify( this.props.attributes ) }></div>
-	}
-}
+
+export const FrontendBlockNode = () => {
+  return (
+    <div className={this.props.className}
```

Comment:
You cannot use `this` outside of a Component, this causes crashes... Instead you probably need to add `props` in the params